qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -126,6 +136,11 @@ def default_values(self) -> List[DefaultCalValue]:
             on the tuples.
         """
 
+    @property
+    def parameters_without_schedule(self) -> Set:
```

Comment:
If we can hide unattached parameter names from users we can remove this property and let Calibrations directly access `lib.__parameters_without_schedule__` (context: I want to minimize public attributes now we need to maintain according to the deprecation policy).

Reply:
Agreed. Done here: [7d8a6ed](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1202/commits/7d8a6edc8ab87ee912f47e9406eac9292658680c)